src/pages/quiz/Quiz.constants.ts

```diff
@@ -0,0 +1,32 @@
+import { Question, QuestionTypesEnum } from '~/types'
+
+export const countPoints = (
+  questions: Question[],
+  answers: Record<string, string | string[]>
+) => {
+  const result = questions.map((item) => {
+    const userAnswer = answers[item._id]
+    if (!userAnswer) return false
+
+    if (
+      item.type === QuestionTypesEnum.MultipleChoice &&
```

Comment:
move this condition to separate variable

Reply:
Thanks! Fixed